test/topology_experimental_raft/test_tablets.py

```diff
@@ -1296,3 +1315,186 @@ async def test_tablet_storage_freeing(manager: ManagerClient):
     logger.info("Verify that the table's disk usage on first node shrunk by about half.")
     size_after = await manager.server_get_sstables_disk_usage(servers[0].server_id, "test", "test")
     assert size_before * 0.33 < size_after < size_before * 0.66
+
+
+async def create_cluster(manager: ManagerClient, num_dcs: int, num_racks: int, nodes_per_rack: int) -> dict[ServerNum, ServerInfo]:
+    logger.debug(f"Creating cluster: num_dcs={num_dcs} num_racks={num_racks} nodes_per_rack={nodes_per_rack}")
+    servers: dict[ServerNum, ServerInfo] = dict()
+    for dc in range(1, num_dcs + 1):
+        for rack in range(1, num_racks + 1):
+            rack_servers = await manager.servers_add(nodes_per_rack, property_file={"dc": f"dc{dc}", "rack": f"rack{rack}"})
+            for s in rack_servers:
+                servers[s.server_id] = s
+    logger.debug(f"Created servers={list(servers.values())}")
+    return servers
+
+
+async def create_and_populate_table(manager: ManagerClient, ks: str = "ks", table: str = "test", rf: int = 3, initial_tablets: int = 64, num_keys: int = 0) -> tuple[str, str, int, int]:
```

Comment:
I would use None for default value for num_keys, it better indicates that the value is gonna be rewritten.

Reply:
okay. I actually coded it this way at first and it seemed overly engineered to me, but I can live with that.

Reply:
fixed in next version